etc/adminhtml/system.xml

```diff
@@ -75,6 +75,12 @@
                     <source_model>Balancepay\Balancepay\Model\Adminhtml\Source\AllowedPaymentMethods</source_model>
                     <config_path>payment/balancepay/allowed_payment_methods</config_path>
                 </field>
+                <field id="net_terms_allowed_payment_methods" translate="label" type="multiselect" sortOrder="80" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
```

Comment:
How is this related to this PR?

Reply:
This is related to ME-51, I forgot to remove related changes from this PR.